### DISCUSSION TITLE
Add a Twilio / Iot / Microvisor Product Icon in Paste Icon Libraries.

### DISCUSSION THREAD

**Discussion by ishah-twilio** [twilio-labs/paste#2122] [Request: Icon] [answered]
Hi Team

I like to request to add Twilio / IoT / Microvisor Product Icon in the paste ICON libraries.

Please let me know what information do you need. Icon is already created and in marketing library.

Thank you

Ishita

**Comment by TheSisb** [comment]
Hi @ishah-twilio,

I've created a ticket into our backlog to add this icon. We will update this discussion thread when the work is completed. If this icon is needed urgently, please do let us know and we will do the best we can to accommodate. Otherwise you can expect it in Figma within a couple weeks, and in code shortly after that.

**Comment by loreina** [comment, marked as answer]
Hey @ishah-twilio,

The Microvisor icon is now available in Figma. There's also a pull request open to add it to code (#2141) and it should be available in our next release.
